test/clj/rems/api/test_resources.clj

```diff
@@ -130,7 +130,32 @@
           (let [result (create-resource "organization-owner1" "organization1")
                 id (:id result)]
             (is (true? (:success result)))
-            (is id)))
+            (is id)
+
+            (testing "fetch using correct organization owner"
+              (let [resource (-> (request :get (str "/api/resources/" id))
+                                 (authenticate api-key "organization-owner1")
+                                 handler
+                                 assert-response-is-ok
+                                 read-body)]
+                (is resource)
+                (is (= [licid] (map :id (:licenses resource))))))
+
+            (testing "fetch using incorrect organization owner"
+              (let [response (-> (request :get (str "/api/resources/" id))
+                                 (authenticate api-key "organization-owner2")
+                                 handler)]
+                (is (response-is-forbidden? response))
+                (is (= "forbidden" (read-body response)))))
```

Comment:
I'd rather have the API return empty so it doesn't leak information about resources etc. This is of course an API design choice.

Reply:
I'll change it to not-found, as proposed by Joel, if that's ok?

Reply:
Done.